cli/src/main.rs

```diff
@@ -541,19 +544,19 @@ fn send_request(matches: &ArgMatches<'_>, call: TrustedCallSigned) -> Option<Vec
 	decoder.register_type_size::<Hash>("H256").unwrap();
 
 	loop {
-		let ret: ProposedSidechainBlockArgs = _chain_api
-			.wait_for_event::<ProposedSidechainBlockArgs>(
+		let ret: ProcessedParentchainBlockArgs = _chain_api
+			.wait_for_event::<ProcessedParentchainBlockArgs>(
```

Comment:
So here we waited for `BlockConfirmed` args. Did block confirm change over time? If not, I am not 100% convinced that this solves the issue.

Reply:
I'm honestly not 100% sure what the previous problem was, but I think it might have been partially fixed by cleaning up the Confirmations on the worker side th PR #501 and now fixing the Block listening event... But the tests pass now without a Timeout. Is that not convincing enough?

Reply:
If they pass consistently, I am convinced. :D